services/apps/alcs/src/portal/pdf-generation/generate-submission-document.service.ts

```diff
@@ -379,4 +384,12 @@ export class GenerateSubmissionDocumentService {
 
     return pdfData;
   }
+
+  private formatPid(pid?: string | null) {
+    const matches = pid?.match(/(.{3})(.{3})(.{3})/);
```

Comment:
Question: I noticed at some point that there are 7-digit PID's in the database. Do we need to match those as well?

Reply:
There shouldn't be any, all PIDs are 9 digits. I'll take a look.

Reply:
Good catch! Update the code to support any length of PIDs.